examples/gluon_alexnet/gluon_hybrid_alexnet.py

```diff
@@ -10,7 +10,7 @@
 
 from mxnet.gluon import nn
 from mxnet.gluon.block import HybridBlock
-from mms.model_service.gluon_vision_service import GluonVisionService
+from ..template.gluon_base_service import GluonBaseService
```

Comment:
template won't be part file path in model archive.

We can assume gluon_base_service.py directly in the model directory.

Reply:
It should be a part of this file, otherwise there would be errors in this file, albeit its a example code.

Reply:
User will copy files under template to /tmp/crepe, not the template folder
The files in /tmp/crepe will be: /tmp/crepe/gluon_vision.service.py,
So the import should be:

from .gluon_base_service import GluonBaseService

Reply:
True. That's how we test it, but having .gluon_base_service would mislead people going through this code because that file doesn't exist in this folder. People reading python code would understand that this service file is in the ```template``` folder. Otherwise, the code is not correct. If we want a working example, we should have proper references to the module. This is similar to the base service code using mxnet_utils. The users could have all the util files also in the same place.